public/assets/frontend-layouts.css

```
@@ -52,9 +52,13 @@ body[class*="layout-"] .book-breadcrumb .breadcrumb-item {
   line-height: 1.4;
 }
 
-body[class*="layout-"] .book-breadcrumb .breadcrumb-item + .breadcrumb-item::before {
+/* Breadcrumb separator for EVERY breadcrumb (book detail, catalogue, CMS…):
+   the catalogue `<ol class="breadcrumb">` is not inside `.book-breadcrumb`, so
+   it had no divider. currentColor adapts to any background — dark text on the
+   book page, white text on the coloured catalogue hero. */
+body[class*="layout-"] .breadcrumb .breadcrumb-item + .breadcrumb-item::before {
   margin-right: .4rem;
-  color: color-mix(in srgb, var(--text-color) 38%, var(--light-bg));
+  color: color-mix(in srgb, currentColor 55%, transparent);
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)